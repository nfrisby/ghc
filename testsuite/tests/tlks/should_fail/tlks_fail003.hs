{-# LANGUAGE TopLevelKindSignatures, NamedWildCards #-}

module TLKS_Fail003 where

type T :: _w
data T = MkT
