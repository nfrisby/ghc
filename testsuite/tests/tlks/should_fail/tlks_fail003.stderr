
tlks_fail003.hs:5:11: error:
    Wildcard ‘_w’ not allowed
      in the top-level kind signature for T
